src/Service.Tests/dab-config.PostgreSql.json

```diff
@@ -2037,6 +2037,39 @@
         "bkname": "name"
       }
     },
+    "BooksSold": {
+      "source": {
+        "object": "books_sold",
+        "type": "table"
+      },
+      "graphql": {
+        "enabled": true,
+        "type": {
+          "singular": "books_sold",
+          "plural": "books_solds"
+        }
+      },
+      "rest": {
+        "enabled": true,
+        "methods": [
+          "get",
+          "post",
+          "put",
+          "patch",
+          "delete"
+        ]
```

Comment:
same comment from MSSQL about not including this section 'methods' if it doesn't cause issues.